networks/basic-rust.yaml

```diff
@@ -23,3 +23,4 @@ spec:
       go: {}
   monitoring:
     namespaced: true
+  ttlSeconds: 3600
```

Comment:
I think it might be better to update the TTL that we send into the driver in the Makefile:
```
# TTL for cleaning up network after tests (defaults to 8 hours)
HERMETIC_TTL ?= 28800
```
Can change it to 3600 there and that will propagate to the Keramik network. That way the network YAML remains simple and can be reused anywhere.

Reply:
Yeah, makes sense, updated!